controllers/secret.go

```diff
@@ -33,19 +35,47 @@ const (
 
 // reconcileKubeConfigSecret will ensure that the kubeconfig Secret is present and up to date.
 func (r *OperatorPipelineReconciler) reconcileKubeConfigSecret(meta metav1.ObjectMeta) error {
+	pipeline := types.NamespacedName{
+		Namespace: meta.Namespace,
+		Name:      meta.Name,
+	}
+	instance := &certv1alpha1.OperatorPipeline{}
```

Comment:
how about `pipeline` for the var name? `instance` is a bit too generic.

Reply:
already used pipeline for the NamespacedName.. pipeline and pipelineInstance ? or pipelineName and pipeline ?

Reply:
or opPipeline